XamarinSamples/DynamoDB/readme.md

```diff
@@ -0,0 +1,9 @@
+# AWS SDK for Dot Net - Xamarin DynamoDB Sample
+
+  
+
+##### Configure the DynamoDB Sample
+
+#####Issues may be encountered 
```

Comment:
Add a space between # and Issues: "#####Issues may be encountered"  >>> "##### Issues may be encountered"

Reply:
Changes according to the comments are done.